src/app/CMakeLists.txt

```diff
@@ -155,6 +155,16 @@ if (WEBUI)
     target_link_libraries(qbt_app PRIVATE qbt_webui)
 endif()
 
+if ((UNIX AND (NOT APPLE)) OR MINGW OR MSYS)
+    add_custom_target(strip_build
+        COMMAND "${CMAKE_COMMAND}" -E "$<IF:$<BOOL:${STACKTRACE}>,echo,echo_append>"
+            "$<IF:$<BOOL:${STACKTRACE}>,WARNING: the STACKTRACE feature is enabled$<COMMA> but stripping the executable will make stacktraces useless.,>"
```

Comment:
Offtopic:
As soon as I see these CMake horrors again, I'm increasingly dejected that it's becoming the standard build system. IMO, "Modern C++ build system in 21th century" should be based on a normal scripting programming language.